rotkehlchen/tests/api/test_aave.py

```diff
@@ -102,43 +102,22 @@ def mock_token_balances(addresses: 'Sequence[ChecksumEvmAddress]') -> TokenBalan
             'lending': {
                 A_DAI.identifier: {
                     'balance': Balance(
-                        amount=FVal('25319.644933920807083535'),
-                        usd_value=FVal('37979.4674008812106253025'),
+                        amount=FVal('21528.498187325929022298'),
```

Comment:
This (and other files with similar changes) changed due to re-recording VCR?

Reply:
yes, values got updated